Package.resolved

```diff
@@ -0,0 +1,23 @@
+{
+  "pins" : [
+    {
+      "identity" : "mcpp",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/zeroc-ice/mcpp.git",
+      "state" : {
+        "branch" : "master",
+        "revision" : "bfd4b159d9428c983e2abb48c8d344003a019caf"
```

Comment:
We should use a label.

Reply:
We can once we tag a new `mcpp` release.